cardano-node/README.md

```diff
@@ -81,33 +81,31 @@ A CLI utility to support a variety of key material operations (genesis, migratio
 
 The general synopsis is as follows:
```

Comment:
This commit should be removed and put in a separate PR.

Reply:
Yes, sure, that's the intent.